content/docs/examples/advanced-gateways/ingress-certmgr/index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Securing Kubernetes Ingress with Cert-Manager
+description: Demonstrates how to obtain "Let's Encrypt" TLS certificates for Kubernetes Ingress automatically using Cert-Manager.
+weight: 70
+keywords: [traffic-management,ingress,https,cert-manager,acme,sds]
+---
+
```

Comment:
Need a intro `##`

Reply:
Looks like all other examples start with intro test right away, without any heading -- kept consistent with the rest (let me know if I misunderstood your comment).

Reply:
Yeah, no need for `#intro` section.